substrate/client/network/src/litep2p/mod.rs

```diff
@@ -557,6 +554,12 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkBackend<B, H> for Litep2pNetworkBac
 			.with_libp2p_ping(ping_config)
 			.with_libp2p_identify(identify_config)
 			.with_libp2p_kademlia(kademlia_config)
+			.with_connection_limits(
+				// By default litep2p accepts only two connections per peer.
```

Comment:
Why do we need multiple connections per peer?

Reply:
Litep2p accepts at most 2 connections with the remote peer to handle cases when both peers dial each other simultaneously. For libp2p, we configure this manually to `2 connections` for similar reasons:
https://github.com/paritytech/polkadot-sdk/blob/d55b09319e3b950be053dc8d5d280f5cecfb7834/substrate/client/network/src/lib.rs#L294-L301